CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ### Fixes
 - Fix wrong url in the dbt docs overview homepage ([#4442](https://github.com/dbt-labs/dbt-core/pull/4442))
-- Fix redefined status param of SQLQueryStatus to typecheck the string which passes on `._message` value of `AdapterResponse` or the `str` value sent by adapter plugin.  ([#4463](https://github.com/dbt-labs/dbt-core/pull/4463#issuecomment-990174166)) 
+- Fix redefined status param of SQLQueryStatus to typecheck the string which passes on `._message` value of `AdapterResponse` or the `str` value sent by adapter plugin.  ([#4463](https://github.com/dbt-labs/dbt-core/pull/4463#issuecomment-990174166))
+- Fix missing data on exposures in docs ([#4467](https://github.com/dbt-labs/dbt-core/issues/4467))
```

Comment:
Historically, we've placed `dbt-docs` changes in their own section, and included the links back to the `dbt-docs` PRs:
```
### Docs
- Fix bug with missing exposure details ([docs#224](https://github.com/dbt-labs/dbt-docs/issues/224), [docs#228](https://github.com/dbt-labs/dbt-docs/pull/228))
- Enabled `markedProvider` built-in sanitization ([docs#227](https://github.com/dbt-labs/dbt-docs/issues/227))
```
Not a blocker to merging this

Reply:
Easy enough fix!